Zonk_2.0/GAME_LOGIC.c

```diff
@@ -0,0 +1,337 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <malloc.h>
+#include <windows.h>
+#include <stdbool.h>
+#include "GAME_CONST.h"
+
+/*������� �������� � �������� ��������� �������� ���������. ���� ������ ���������� - ���������� ������, � ��������� ������ - ����*/
+bool checkWin(int result)
+{
+	return result > WINNING_RESULT - 1;
+}
+
+/*������ ���������� �������� ����� �� 1 �� 6*/
+int generatingRandomDice()
+{
+	int number;
+	number = rand() % 5 + 1;
+
+	return number;
+}
+
+
+/*������ �������� � �������� ��������� ���������� ���������������� ������ � ���������� ��������� �� ������ ��������� ������*/
+void fillRandomDices(int unusedDice, int(*fun)(), int* dices)
```

Comment:
1. Order of function arguments. As I think and have often seen, in similar situations data source (there is an array) stays in first place. Then let source's size will be on second. Finally, functions' pointers' go in last order.
2. Naming. Arguments' names does not clearly say anything about what this function does. `unusedDice` means that function will handle this value in specific way or it is an array size? `fun`, really?))
3. Misunderstanding. Function doesn't return value. Return value or correct comment.